userguide/tasks.rst

```diff
@@ -37,6 +37,8 @@ Each of these tasks is described in more detail in this section.
    automatically-created task. :ref:`S.M.A.R.T. Tests` and
    :ref:`Periodic Snapshot Tasks` must be set up manually.
 
+.. note:: Spaces cannot be used in the task scheduler.
```

Comment:
Probably would be best to avoid having two note boxes right next to each other.

Reply:
Ya that crossed my mind. I couldn't really think of another solution for such a small aside about the scheduler. Do you have suggestions?

Reply:
This is probably not specific enough.  Spaces cannot be used at all, or only in certain fields, or not in times?  This note might be better with the specific fields where spaces are not allowed, or integrated into the descriptions of each field that does not allow spaces.